### PR TITLE
[Communications] communications linting fixes

### PR DESCRIPTION
**Pull request by annatisch**

### CODE REVIEW

File: sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py [hunk 1]
```diff
@@ -352,28 +347,27 @@ def list_available_localities(
     @distributed_trace
     def list_available_offerings(
         self,
-        country_code,  # type: str
-        **kwargs
-    ):
-        # type: (...) -> ItemPaged[PhoneNumberOffering]
+        country_code: str,
+        **kwargs: Any
```
Comment (xiangyan99) [inline sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py:351]: Do we want to use something like

*, phone_number_type

?

**Review by xiangyan99 — APPROVED** [review]
I see we use **kwargs rather than explicit key word arguments in the function signature.

I will let you make the call if we want to update it or not.

File: sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py (hunk 1 above)
Comment (annatisch) [reply sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py:351]: Yeah, we should bump all these to proper keyword-only....
However this SDK is currently in beta, with a few new features on their way through preview - so I think I will ask the service team to make those changes as part of their next GA cycle.

**Review by sarkar-rajarshi — APPROVED** [review]
Looks good to me @annatisch

**Pull request merged**